setup.py

```diff
@@ -7,6 +7,14 @@
                      'correctly.\n')
     sys.exit(0)
 
+if sys.platform == 'darwin':
+    # You *need* to use clang and clang++ for ParmEd extensions on a Mac;
+    # Anaconda does annoying stuff that breaks this, since their distutils
+    # automatically tries to use "gcc", which would conflict with the MacPorts
+    # gcc... sigh.
+    os.environ['CXX'] = 'clang++'
```

Comment:
is clang++ always available on Mac?

have you tried using Cython to call C/C++ function in python? It seems to me that cython supports different compilers well. (but I am not sure about ParmEd's case).